packages/core/src/v3/errors.ts

```
@@ -149,6 +150,58 @@ export function sanitizeError(error: TaskRunError): TaskRunError {
+        case "TASK_EXECUTION_ABORTED":
+        case "TASK_EXECUTION_FAILED":
+        case "TASK_RUN_CRASHED":
+        case "TASK_RUN_HEARTBEAT_TIMEOUT":
+        // TODO: check we really want to retry here, these could be oom errors
+        case "TASK_PROCESS_EXITED_WITH_NON_ZERO_CODE":
+          return true;
+
+        default:
+          assertExhaustive(error.code);
+      }
+    }
```

Biome

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)